### ISSUE TITLE
Feature Request: Custom model selection

### ISSUE THREAD

**Issue by Jorvan758** [divamgupta/diffusionbee-stable-diffusion-ui#149]
This app is awesome, but this is an important lacking feature. Some of us would love to re-train the model to make it able to generate ourselves (for which there are easy options, like some public google colab notebooks). I think that it would be fairly easy to enable, so I hope it's added in the next version

Anyway, thanks for the app and keep on the great work 🙌

**Comment by JackHamer09** [comment]
+1

**Comment by Maki9009** [comment]
+1000 please

**Comment by camilovietnam** [comment]
Yes, from all the SD apps I have tried, Diffusion Bee comes out as winner, I wish I didn't have to install other apps to use different models!

**Comment by Maki9009** [comment]
> Yes, from all the SD apps I have tried, Diffusion Bee comes out as winner, I wish I didn't have to install other apps to use different models!
> 
> I tried to trick Bee by downloading another model and putting it as the .ckpt file but it must be running integrity checks because it just ignored the model and proceeded to download SD from the Internet

what other apps have you tired on Mac? have you Managed to Install Automatics Repo on the M1?

**Comment by camilovietnam** [comment]
> > Yes, from all the SD apps I have tried, Diffusion Bee comes out as winner, I wish I didn't have to install other apps to use different models!
> > I tried to trick Bee by downloading another model and putting it as the .ckpt file but it must be running integrity checks because it just ignored the model and proceeded to download SD from the Internet
> 
> what other apps have you tired on Mac? have you Managed to Install Automatics Repo on the M1?

AUTO1111? Yes, here is a screenshot:

![auto](https://user-images.githubusercontent.com/49581907/193428713-f3ecadbd-6072-4b3f-be19-afac0e9e1750.jpg)

But this one is much slower than Diffusion Bee, and sometimes it just generates a black image. I think it's possible there are some libraries missing and it's just using CPU instead of GPU, I remember reading something in the log about this, but I don't know if that has any real effect on it. I've just noticed that results are better (as in, faster) with Diff Bee.

I think I was also able to install Invoke AI, here is a screenshot:
<img width="1438" alt="Screen Shot 2022-10-01 at 23 26 07" src="https://user-images.githubusercontent.com/49581907/193428920-599d9614-bcfb-4484-ac83-6ad21b53fa32.png">

But as I said, I prefer Diffusion Bee because it seems to be the fastest of all the ones I have tried, not to mention it now includes an option to generate multiple images, and a History tab, which I absolutely love.
